src/tests/access/test_dual_accesscontrol.cairo

```diff
@@ -68,35 +63,37 @@ fn test_dual_supports_interface() {
 }
 
 #[test]
+#[ignore] // REASON: inconsistent ENTRYPOINT_NOT_FOUND panic message
```

Comment:
We need to update this messages, and fix the ones in Ownable as well. We merged the Ownable PR with the ignored tests inconsistencies since that PR was blocking presets.

Reply:
Updated the error messages